src/containers/DataTable/Table.js

```diff
@@ -78,21 +78,26 @@ const TableCell = ({ obj, header, idx, showJson, jsonTheme }) => {
 		cell = <>{icon} {cell}</>;
 	}
 
+	const style = {
+		whiteSpace: "nowrap",
+		maxWidth: "260px",
+		textOverflow: "ellipsis",
+		overflow: "hidden"
+	}
+
 	return idx === 0 ? (
 			<th
 				className="data-table-th"
 				scope="row"
-				style={{
-					whiteSpace: "nowrap",
-					maxWidth: "40rem",
-					textOverflow: "ellipsis",
-					overflow: "hidden" 
-				}}
+				style={ style }
```

Comment:
You shouldn't add maxWidth and ellipsis here. Add it only for links (line 50) and for common text (line 75).

Reply:
Are you sure about line 75 ? It's `else cell = obj[header.key] ? obj[header.key] : "-";` on it

Reply:
Oh, I see now! I was not right, sorry

Reply:
@t-pyrope yes, I'm sure. You should wrap `obj[header.key]` and `"-"` into some span and add styles there. Otherwise if you add styles on line 92 you will apply this styles to all other components (customComponent and datetime as well, but we don't want them to be limited by width)